### PR TITLE
Update EVALUATION.md

### PR DESCRIPTION
**Pull request by kjetilkjeka**

### CODE REVIEW

**Comment by kjetilkjeka** [comment]
cc: @klasbo

File: EVALUATION.md [hunk 1]
```diff
@@ -42,7 +42,35 @@ You will need to consult the [specification](https://github.com/TTK4145/Project#
...
+     - Comments should not be repetition of code.
+     - Comments should generally express intent on a different abstraction level than code.
+     
+##### Modules
+ - Modules should minimize accessibility to their internals.
+     - Programmatically enforce this where possible (Erlang, Rust, etc)
+     - Enforce this through conventions if there are no language features to enable this (Python, etc)
+ - Modules should hide their implementation details.
+ - Module interfaces should represent a consistent abstraction.
+ - The module names should fit weel with the module interface.
```
Comment (klasbo) [inline EVALUATION.md]: Spelling weel -> well

File: EVALUATION.md [hunk 2]
```diff
@@ -42,7 +42,35 @@ You will need to consult the [specification](https://github.com/TTK4145/Project#
...
+ - Module interfaces should be simple yet complete.
+     - Avoid interfaces that is hard to use correctly.
```
Comment (klasbo) [inline EVALUATION.md]: Grammar: is -> are

File: EVALUATION.md [hunk 3]
```diff
@@ -42,7 +42,35 @@ You will need to consult the [specification](https://github.com/TTK4145/Project#
...
+     
+##### Routines
+ - A routine should have a concrete purpose (do one thing well).
+     - The word "and" in a routine name usually means that something is "wrong".
+ - The routine name should indicate its purpose.
+ - Routines should be simple enough to veirfy from inspection that it fulfill its purpose without doing anything more.
```
Comment (klasbo) [inline EVALUATION.md]: Spelling: fulfill -> fulfills

File: EVALUATION.md [hunk 4]
```diff
@@ -42,7 +42,35 @@ You will need to consult the [specification](https://github.com/TTK4145/Project#
...
+### Evaluation Criteria
+#### Personal skills
+ - You should be able to evaluate code quality based on the [code evaluation criterias](https://github.com/TTK4145/Project/blob/master/EVALUATION.md#Code-evaluation)
+ 
+#### Code evaluation
+##### Top level
+ - The entry point (or similar) should document what components/modules the system consists of.
+ - It should be clear how different comonents/modules communicate and depend on each other, and how they depend on external components.
+ - Naming should be consistent over the whole program.
+ - Programming features should be used in a good way.
```
Comment (klasbo) [inline EVALUATION.md]: "Language features"? Or "The programming language should be used in an idiomatic way"?

**Comment by klasbo** [comment]
Should there be something "higher level" about how data flows through the program? Creating simple modules/routines with complex interactions is easy but bad.

**Comment by kjetilkjeka** [comment]
> Should there be something "higher level" about how data flows through the program? Creating simple modules/routines with complex interactions is easy but bad.

Do you think we need anything more than "module interfaces should be simple" in combination with "modules should hide their implementation details"?

I will merge as is and if anything more is needed we can add it later.

**Pull request merged**